documentation/api/query/v4/pql.markdown

```diff
@@ -381,3 +381,32 @@ Combined with aggregate functions, you can effectively rollup of results and agg
 up values, for example to return how many facts exist for each fact name across all facts:
 
     facts[name, count(value)] { certname ~ 'web.*' group by name }
+
+## Paging
+PQL supports restriction of the result set via the standard SQL paging clauses
```

Comment:
Might be worth re-iterating somewhere that an order and offset, doesn't make much sense without an order by. We state this elsewhere, I think in the AST doc, but since we've split this up now might be worth mentioning again.